content/en/about/feature-stages/index.md

```diff
@@ -43,6 +43,7 @@ Below is our list of existing features and their current phases. This informatio
 | Enabling custom filters in Envoy | Alpha
 | CNI container interface | Alpha
 | [Sidecar API](/docs/reference/config/networking/sidecar/) | Beta
+| [DNS Proxying](docs/ops/configuration/traffic-management/dns-proxy/) | Alpha
```

Comment:
Added this as from a earlier code review it went from experimental and should be Alpha for this release